server/services/totalCount.service.js

```diff
@@ -42,6 +44,7 @@ const getTotalCount = async (user) => {
       ...matchObject,
     });
     const transactions = await Transaction.countDocuments(matchObject);
+    const assignedbadges = await AssignedBadge.countDocuments({ userId: ObjectId(user._id) });
```

Comment:
Typo:
```suggestion
    const assignedBadges = await AssignedBadge.countDocuments({ userId: ObjectId(user._id) });
```

Since this would also be returned for the admin, I think we should also add a test for it.